migrations/redesign/message_composer.md

```
@@ -8,7 +8,7 @@ This guide covers the migration for the message composer components in the Strea
 
 - [Overview](#overview)
 - [StreamMessageInput](#streammessageinput)
-- [StreamChatMessageComposer (new)](#streamchatmessagecomposer-new)
+- [StreamMessageComposer (new)](#streamchatmessagecomposer-new)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)